sample/src/androidTest/java/cloudinary/android/sample/UploadWidgetTest.java

```diff
@@ -41,16 +38,13 @@ public static void setup() throws IOException {
     public IntentsTestRule<MainActivity> intentsTestRule = new IntentsTestRule<>(MainActivity.class);
 
     // TODO: Fix UI tests for travis
-    @Ignore
+//    @Ignore
```

Comment:
Please remove

Reply:
Remove the test altogether? Or remove the `@ignore`? This test will fail every travis job until fixed.

Reply:
The test or the annotation? or the comment ? what state do you want this part to be in master?

Reply:
The comment